crates/libs/windows/src/lib.rs

```diff
@@ -10,6 +10,12 @@ Learn more about Rust for Windows here: <https://github.com/microsoft/windows-rs
 #![cfg_attr(windows_raw_dylib, feature(raw_dylib))]
 
 extern crate self as windows;
-mod Windows;
-pub mod core;
+
 pub use Windows::*;
+pub mod core;
+
+#[doc(hidden)]
+mod Windows;
+
+#[doc(hidden)]
+pub mod imp;
```

Comment:
Is there a technical reason why this cannot be

```rust
pub(crate) mod imp;
```

?

Reply:
Yes, there are implementation details that are nevertheless used outside of the `windows` crate because the `windows-bindgen` crate will generate them for non-Windows metadata.